packages/manager/modules/web-universe-components/src/order-cart/order-cart.service.js

```diff
@@ -6,24 +14,46 @@ export default class OrderCartService {
     this.OvhApiOrder = OvhApiOrder;
   }
 
+  /**
```

Comment:
Are you using a convention for documentation block comments?

In my opinion, I don't find useful to repeat the method name and then describe it.

Why not considering something like (template):

```txt
  /**
   * [createNewCart description]
   * @param  {[type]} ovhSubsidiary [description]
   * @return {[type]}               [description]
   */
```

Notice here I'm also mentioning the `@return`.

FYI I'm using the following extension: https://atom.io/packages/docblockr

Reply:
@antleblanc, I'm just applying the auto-generated template from https://atom.io/packages/atom-easy-jsdoc. 

So I've let the method name, and just removed the `@return {type}` line, for those calls which returns $promise, because I don't see much benefits to say to user : `@return Promise<Object>`, as it is binded to the API response.

Let me know your point of view.

Reply:
Well telling that you can chain your method call and not use its value directly without looking at the code is lit 🔥

Reply:
I think we should have an uniformized way to comment our codes :-)

Reply:
You mean like `js-doc` ? 😄 

I agree with you, it would make it easier to have a component library yes

Reply:
Ok, adding the return line